src/View/Components/NativeSelect.php

```diff
@@ -78,4 +82,13 @@ public function getOptionLabel($key, $option)
 
         return data_get($option, $this->optionLabel);
     }
+
+    public function getOptionSubtitle($key, $option, $native = false)
```

Comment:
Simplify this function
Override this function to work in the Select component.

```suggestion
    public function getOptionSubtitle($key, $option)
```